lib/main.dart

```diff
@@ -3,7 +3,7 @@ import 'dart:io';
 
 import 'package:flutter/material.dart';
 import 'routes.dart';
-import 'pages/structure.dart';
+import 'package:sossoldi/pages/onboarding_page.dart';
```

Comment:
It's better to use relative path as mentioned in #43 . Check also the other files commited to update it.

Reply:
Maybe the best choice is to use relative path :'D

Reply:
Thank you! I just made a commit to fix this. 

I think there are others incorrectly named import in the project, maybe its needed a issue for check all files.